packages/react-components/src/components/VideoGallery/utils/videoGalleryLayoutUtils.ts

```diff
@@ -70,15 +70,12 @@ const getOrganizedParticipants = (props: OrganizedParticipantsArgs): OrganizedPa
     maxDominantSpeakers: maxGridParticipants
   }).slice(0, maxGridParticipants);
 
-  const dominantSpeakerToGrid =
-    layout === 'speaker'
-      ? dominantSpeakers && dominantSpeakers[0]
-        ? newGridParticipants.filter((p) => p.userId === dominantSpeakers[0])
-        : [newGridParticipants[0]]
-      : [];
-
-  if (dominantSpeakerToGrid[0]) {
-    newGridParticipants = dominantSpeakerToGrid;
+  if (layout === 'speaker') {
+    if (dominantSpeakers?.[0]) {
+      newGridParticipants = newGridParticipants.filter((p) => p.userId !== dominantSpeakers[0]);
```

Comment:
I think we want to filter here to collect just the dominant speaker thats at the 0th index. right now this will do the reverse where we are in speaker and it will put the dominant speaker in the overflow and everyone else in the grid.

Reply:
great catch! not sure why I chagned that from `===` to `!==`

Reply:
Updated